Directory.Build.props

```diff
@@ -90,8 +90,9 @@
     <MicrosoftApplicationInsightsVersion>2.4.0</MicrosoftApplicationInsightsVersion>
     <MicrosoftAzureCosmosTableVersion>1.0.5</MicrosoftAzureCosmosTableVersion>
     <MicrosoftAzureEventHubsVersion>4.1.0</MicrosoftAzureEventHubsVersion>
-    <MicrosoftAzureStorageBlobVersion>11.1.0</MicrosoftAzureStorageBlobVersion>
-    <MicrosoftAzureStorageQueueVersion>11.1.0</MicrosoftAzureStorageQueueVersion>
+    <AzureIdentityVersion>1.1.1</AzureIdentityVersion>
```

Comment:
Only `Azure.Core` should be added as a new dependency, `Azure.Identity` would pull in a whole bunch of additional dependencies that are not needed to support the AAD options. Only users that actually will set the `options.TokenCredential` field value will need to add the additional dependencies.

Reply:
Yeah, the base `TokenCredential` is what is the required type. If the user will use any of the specialised ones they can add the Identity package.

Reply:
`DefaultAzureCredential` is used in tests.

Reply:
Then only that test project needs Azure.Identity